src/OrchardCore.Modules/OrchardCore.Users/Controllers/RegistrationController.cs

```diff
@@ -74,12 +70,7 @@ public async Task<IActionResult> Register(RegisterViewModel model, string return
                 return NotFound();
             }
 
-            if (string.IsNullOrEmpty(model.Email))
-            {
-                ModelState.AddModelError("Email", S["Email is required."]);
-            }
-
-            if (_emailAddressValidator.Validate(model.Email))
```

Comment:
The `IEmailValidator` is useful for checking the email syntax

Reply:
That is already done in the `RegisterViewModel`: https://github.com/OrchardCMS/OrchardCore/blob/40d3f65663c8582a9c76c59d4f5bdfaf781d0bea/src/OrchardCore.Modules/OrchardCore.Users/ViewModels/RegisterViewModel.cs#L35